jbmc/src/java_bytecode/java_multi_path_symex_only_checker.cpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************\
+
+Module: Goto Checker using Bounded Model Checking for Java
+
+Author: Daniel Kroening, Peter Schrammel
+
+ \*******************************************************************/
+
+/// \file
+/// Goto Checker using Bounded Model Checking for Java
+
+#include "java_multi_path_symex_only_checker.h"
+
+#include <java_bytecode/java_enum_static_init_unwind_handler.h>
+
+java_multi_path_symex_only_checkert::java_multi_path_symex_only_checkert(
+  const optionst &options,
+  ui_message_handlert &ui_message_handler,
+  abstract_goto_modelt &goto_model)
+  : multi_path_symex_only_checkert(options, ui_message_handler, goto_model)
+{
+  // unwinds <clinit> loops to number of enum elements
+  if(options.get_bool_option("java-unwind-enum-static"))
```

Comment:
Maybe factor this code with the identical code in jbmc?

Reply:
Factored out into separate file. The instance of the code in jbmc_parse_options will disappear with the `do_language_agnostic_bmc` call.